src/MyPlot/events/MyPlotPlayerLeavePlotEvent.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace MyPlot\events;
+
+use MyPlot\MyPlot;
+use MyPlot\Plot;
+use pocketmine\Player;
+
+class MyPlotPlayerLeavePlotEvent extends MyPlotPlotEvent{
+	public static $handlerList = null;
+	/** @var Player */
+	private $player;
+
+	/**
+	 * PlotEnterEvent constructor.
```

Comment:
Fix the class name here too

Reply:
DIY. It is your repository, i was just trying to appy fixes/extensions